### PR TITLE
Adding optional chaining operator example

### PR DESCRIPTION
**Pull request by SphinxKnight**
This adds a basic interactive example for a new (stage 3) operator

- Spec: https://github.com/tc39/proposal-optional-chaining/
- Target page on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
- Bug for implementation with DDN: https://bugzilla.mozilla.org/show_bug.cgi?id=1566143

### CODE REVIEW

File: live-examples/js-examples/expressions/expressions-optionalchainingoperator.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<pre>
+<code id="static-js">const adventurer = {
+  name: 'Alice',
+  cat: {
+    name:'Dinah'
```
Comment (wbamberg) [inline live-examples/js-examples/expressions/expressions-optionalchainingoperator.html]: Should have a space between key and value: `name: 'Dinah'`

**Comment by SphinxKnight** [comment]
@wbamberg no this does not seem to be implemented yet. I should have read the policy on the JavaScript section (advanced stage in TC39 + at least 1 impl) before jumping on this one.
I'll merge the changes and then we can close this PR (and reopen it one day when a browser implements it).

Thanks for the review :)

File: live-examples/js-examples/expressions/expressions-optionalchainingoperator.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+};
+
+const dogName = adventurer.dog?.name;
+console.log(dogName);
+// expected output: undefined
+
+console.log(adventurer.someNonExistingMethod?.())
+// expected output: undefined
+</code>
+</pre>
```
[the remark this answers is not in the document]
Comment (SphinxKnight) [reply live-examples/js-examples/expressions/expressions-optionalchainingoperator.html]: Fixed, thanks!

**Comment by wbamberg** [comment]
I've thought about it, and since the docs are there, ad it seems quite stable, let's merge it. We can add it to the docs page when browser support is better.

**Pull request merged**